src/pages/product-details/index.js

```diff
@@ -1,3 +1,3 @@
-import ProductDetail from './product-details-page';
+import ProductDetails from './product-details-page';
```

Comment:
Please, rename the file to the `product-details`

Reply:
Renamed

Reply:
renamed